VirtualDevelopmentEnvironment.md

```diff
@@ -1,41 +1,50 @@
-## Virtual development environment
+# Virtual Development Environment
 
-It's recomended to use [Vagrant][] and [VirtualBox][] in order to achieve consistent development environment across all contributors.
+It's recomended to use [Vagrant][] and [VirtualBox][] in order to achieve
+consistent development environment across all contributors.
 
-### Installation
+## Installation
 
-- Download and install latest [VirtualBox][]
-- Download and install latest [Vagrant][]
-- Clone GitHub repo:
-    
-    ```
+1.  Download and install latest [VirtualBox][].
+2.  Download and install latest [Vagrant][].
+3.  Clone GitHub repo:
+
+    ```shell
     $ git clone git@github.com:apiaryio/dredd.git
```

Comment:
I'd change this to use `https://`. Git over https has less upfront expectations. Using SSH as you have, it requires users to already have setup a GitHub account and to have an associated SSH keys. The HTTPS clone URL works for people who don't have an account or haven't setup an SSH key. Or even when a user is behind some firewall preventing SSH access (I know many companies who do this).

Example from those who don't:

```shell
$ git clone git@github.com:apiaryio/dredd.git
Cloning into 'dredd'...
The authenticity of host 'github.com (192.30.252.131)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'github.com,192.30.252.131' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

```shell
$ git clone https://github.com/apiaryio/dredd
Cloning into 'dredd'...
remote: Counting objects: 4523, done.
remote: Compressing objects: 100% (30/30), done.
remote: Total 4523 (delta 9), reused 0 (delta 0), pack-reused 4492
Receiving objects: 100% (4523/4523), 1.53 MiB | 1.22 MiB/s, done.
Resolving deltas: 100% (3053/3053), done.
Checking connectivity... done.
```